soc/bouffalolab/bl6/rodata.ld

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2021-2024 ATL Electronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <autoconf.h>
+
+#if defined(CONFIG_SOC_SERIES_BL6)
```

Comment:
just include this file if option is enabled

Reply:
Already present on CMakeList.txt